### PR TITLE
Get rid of zookeeper

### PR DESCRIPTION
**Pull request by Haarolean**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalClusterState.java [hunk 1]
```diff
@@ -40,7 +38,6 @@ public InternalClusterState(KafkaCluster cluster, MetricsCache.Metrics metrics)
         .orElse(null);
     topicCount = metrics.getTopicDescriptions().size();
     brokerCount = metrics.getClusterDescription().getNodes().size();
-    zooKeeperStatus = ClusterUtil.convertToIntServerStatus(metrics.getZkStatus().getStatus());
```
Comment (iliax) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalClusterState.java:43]: ClusterUtil.convertToIntServerStatus method can be removed.

File: README.md [hunk 2]
```diff
@@ -161,7 +159,6 @@ For example, if you want to use an environment variable to set the `name` parame
 |`KAFKA_ADMIN-CLIENT-TIMEOUT` | Kafka API timeout in ms. Default: `30000`
 |`KAFKA_CLUSTERS_0_NAME` | Cluster name
 |`KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS` 	|Address where to connect 
-|`KAFKA_CLUSTERS_0_ZOOKEEPER` 	| Zookeeper service address
```
Comment (iliax) [inline README.md:164]: I would also add add deprecated flag to `ClusterStats.zooKeeperStatus` in kafka-ui-api.yaml

**Review by iliax — APPROVED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalClusterState.java (hunk 1 above)
Comment (Haarolean) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalClusterState.java:43]: done

**Review by germanosin — APPROVED** [review]

**Review by iliax — APPROVED** [review]

**Comment by Kapkiai** [comment]
Hi Haarlonean,
Also remove zookeeper on UI.
Regards

**Comment by Haarolean** [comment]
@Kapkiai hey, #1663

**Pull request merged**